Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use num_bigint::{BigUint, ToBigUint};
+use std::io;
+
+// convert input string to BigUint
+fn read_parse(inp: String) -> BigUint {
```

Comment:
Don't need to pass a `String` object, which copies the object (but not the &str data) on the stack). A `&String` reference is enough. Actually, don't need a `&String` reference, which sometimes means allocating an extra object, since a `&str` is enough.
Rule of thumb: use `&str` if you can.
`String` is an object that wraps a heap-allocated &str. Use `String` if you can't use `&str`, eg. if a function returns a String and the returned object needs to own the &str inside.

Reply:
done